telegram/ext/filters.py

```diff
@@ -324,6 +324,65 @@ def filter(self, message):
 
     group = _Group()
 
+    class user(BaseFilter):
+        """Filters messages to allow only those which are from specified user ID
+
+        Args:
+            user_id(Optional[int]): which user ID to allow through. Required if username is not
+                specified
+            username(Optional[str]): which username to allow through. Required if user_id is not
+                specified
+
```

Comment:
1. Documentation should be explicit on the fact that one and only one of `user_id` & `username` must be specified.
2. please add a period at the end of your sentences.
3. We use the extension of pep8 allowing 99 chars per line, so you can have the lines in matching size.